src/CoreBundle/Controller/ChameleonController.php

```diff
@@ -132,6 +138,7 @@ public function __construct(
         RequestStack $requestStack,
         EventDispatcherInterface $eventDispatcher,
         PortalDomainServiceInterface $portalDomainService,
+        ResourceCollectorInterface $resourceCollector,
```

Comment:
I'd like to avoid having to adjust constructor arguments for the controller, as custom implementations will rely on this class instead of the interface, and will then break. Could you use setter injection as we already do it for some other dependencies?

Reply:
ok

Will such things be changed for example for 6.3?